api/src/main/java/org/openmrs/api/UserService.java

```diff
@@ -578,4 +578,13 @@ public List<User> getUsers(String name, List<Role> roles, boolean includeRetired
 	 * @since 2.3.6, 2.4.6, 2.5.4, 2.6.0
 	 */
 	Locale getDefaultLocaleForUser(User user);
+
+	/**
+	 * Retrieves the last login time of the user
```

Comment:
Is the retrieved last login time in milliseconds?

Reply:
No, in Unix Timestamp (Epoch Unix Timestamp)

Reply:
That answer could be in the code instead of github comment. 😊